### PR TITLE
[POC] Fetching Top Participants

### PR DESCRIPTION
**Pull request by lucasmontano**

### CODE REVIEW

**Review by omurilo — APPROVED** [review]
I think the currentColor is the participant returned from API, so I would rename this to participant.

**Comment by lucasmontano** [comment]
@omurilo I'll remove all the code related to it. I was thinking: we can have a simple loop requesting the participants from the extension BE to the API every minute and broadcasting this to the FE.

File: public/viewer.js [hunk 1]
```diff
@@ -1,76 +1,66 @@
...
 }
 
 function setAuth(token) {
-    Object.keys(requests).forEach((req) => {
-        twitch.rig.log('Setting auth headers');
-        requests[req].headers = { 'Authorization': 'Bearer ' + token }
-    });
+  Object.keys(requests).forEach((req) => {
+    twitch.rig.log("Setting auth headers")
+    requests[req].headers = { Authorization: "Bearer " + token }
```
Comment (matheusmelchiades) [inline public/viewer.js]: It Should use template string

File: services/backend.js [hunk 2]
```diff
@@ -14,229 +14,281 @@
...
+      .get("https://montano-twitch-extension.herokuapp.com/")
+      .then(function (response) {
+        // handle success
+        topParticipants = response.data
+      })
+      .catch(function (error) {
+        // handle error
+        console.log(error)
+      })
+      .then(function () {
```
Comment (matheusmelchiades) [inline services/backend.js]: It should to be removed, because it doesn't  nothing

**Pull request merged**